CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 22.9
+This update enhances Shipping Labels with improved accessibility, smarter address validation, and faster performance. We've also added a dedicated Point of Sale tab for quicker access, and POS orders are now filterable in your order list for better organization. Plus, we've optimized assets to reduce the app's size!
```

Comment:
non-blocking nit: feel free to make the final decision!

The POS tab was first introduced in 22.7, but this version makes it more widely available to stores in eligible countries.

> Plus, we've optimized assets to reduce the app's size!

The exclamation mark feels a bit excessive for me 😅 

> We've also added a dedicated Point of Sale tab for quicker access
```suggestion
This update enhances Shipping Labels with improved accessibility, smarter address validation, and faster performance. We’ve also made the dedicated Point of Sale tab more widely available for quicker access, and POS orders are now filterable in your order list for better organization. Plus, we've optimized assets to reduce the app’s size.
```